feature_server/config/scripts/spawn_protect.py

```diff
@@ -2,30 +2,28 @@
 Protects spawned players for a specified amount of seconds.
 
 Maintainer: ?
+Fixed error by kmsi(kmsiapps@gmail.com) : replaced player to hit_player
 """
 
 from pyspades.common import prettify_timespan
 from twisted.internet import reactor
 
 def apply_script(protocol, connection, config):
     spawn_protect_time = config.get('spawn_protect_time', 3.0)
-
+    
     class SpawnProtectConnection(connection):
         spawn_timestamp = None
 
         def on_spawn(self, pos):
             self.spawn_timestamp = reactor.seconds()
             return connection.on_spawn(self, pos)
 
-        def on_hit(self, hit_amount, player, type, grenade):
+        def on_hit(self, hit_amount, hit_player, type, grenade):
             cur_timestamp = reactor.seconds() - spawn_protect_time
             if cur_timestamp < hit_player.spawn_timestamp:
                 timespan = -(cur_timestamp - hit_player.spawn_timestamp)
-                self.send_chat(
-                "%s is spawn-protected for %s." %
-                    (player.name,
-                     prettify_timespan(timespan, True)))
+                self.send_chat( "%s is spawn-protected for %s." %(hit_player.name, prettify_timespan(timespan, True)))
                 return False
-            return connection.on_hit(self, hit_amount, player, type, grenade)
-
-    return protocol, SpawnProtectConnection
+            return connection.on_hit(self, hit_amount, hit_player, type, grenade)
+    
+    return protocol, SpawnProtectConnection
```

Comment:
An empty line as the last line of the file. Not all text editors insert it automatically, although they should. Right now it ends abrubtly.

Basically you know you got it right when you get rid of this little icon in github file view: 
<img width="409" alt="screen shot 2017-01-22 at 8 04 34 pm" src="https://cloud.githubusercontent.com/assets/2031472/22180752/4ecd39a8-e0de-11e6-9a65-d6fd8ed0a85b.png">